<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>Research - Lopatkin Lab</title>
  <meta content="" name="description">
  <meta content="" name="keywords">

  <!-- Favicons -->
  <link href="assets/img/favicon.png" rel="icon">
  <link href="assets/img/apple-touch-icon.png" rel="apple-touch-icon">

  <!-- Google Fonts -->
  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|Roboto:300,300i,400,400i,500,500i,700,700i&display=swap" rel="stylesheet">

  <!-- Vendor CSS Files -->
  <link href="assets/vendor/animate.css/animate.min.css" rel="stylesheet">
  <link href="assets/vendor/aos/aos.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap-icons/bootstrap-icons.css" rel="stylesheet">
  <link href='https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css' rel='stylesheet'>
  <link href="assets/vendor/glightbox/css/glightbox.min.css" rel="stylesheet">
  <link href="assets/vendor/swiper/swiper-bundle.min.css" rel="stylesheet">

  <!-- Template Main CSS File -->
  <link href="assets/css/style.css" rel="stylesheet">

  <!-- =======================================================
  * Template Name: Moderna - v4.8.0
  * Template URL: https://bootstrapmade.com/free-bootstrap-template-corporate-moderna/
  * Author: BootstrapMade.com
  * License: https://bootstrapmade.com/license/
  ======================================================== -->
</head>

<body>

  <!-- ======= Header ======= -->
  <header id="header" class="fixed-top d-flex align-items-center ">
    <div class="container d-flex justify-content-between align-items-center">

      <div class="logo">
        <h1 class="text-light"><a href="index.html"><span>Lopatkin Lab</span></a></h1>
        <!-- Uncomment below if you prefer to use an image logo -->
        <!-- <a href="index.html"><img src="assets/img/logo.png" alt="" class="img-fluid"></a>-->
      </div>

      <nav id="navbar" class="navbar">
        <ul>
          <li><a href="index.html">Home</a></li>
          <li><a href="people.html">People</a></li>
          <li><a href="publications.html">Publications</a></li>
          <li><a href="resources.html">Resources</a></li>
          <li><a class="active" href="research.html">Research</a></li>
          <li><a href="join.html">Join Us</a></li>
        </ul>
        <i class="bi bi-list mobile-nav-toggle"></i>
      </nav><!-- .navbar -->

    </div>
  </header><!-- End Header -->

  <main id="main">

    <!-- ======= Our Services Section ======= -->
    <section class="breadcrumbs">
      <div class="container">

        <div class="d-flex justify-content-between align-items-center">
          <h2>Research</h2>
          <ol>
            <li><a href="index.html">Home</a></li>
            <li>Research</li>
          </ol>
        </div>

      </div>
    </section><!-- End Our Services Section -->

    <!-- ======= Services Section ======= -->
    <section class="why-us section-bg" id="detail">
      <div class="container">
         <center><h4 class="title"><br>What is Horizontal Gene Transfer?</h4></center>

         <div class="row align-items-center">
          <div class="col-lg-4">
              <p class="description">Horizontal gene transfer (HGT) is a naturally occurring phenomenon that allows microbes to transfer large pieces of DNA amongst one another. HGT serves as the primary way that bacteria, and especially pathogens, adapt to new and often stressful environments. More than its clinical relevance, HGT is ubiquitous: it occurs in every unique microbial niche on earth, and acts as a robust tool for cell-cell communication despite highly complex and dynamic environments that are constantly fluctuating in chemical composition and biological diversity. </p>
          </div>

           <div class="col-lg-4">
            <img src="assets/img/research/hgt.png" class="img-fluid" alt="">
          </div>

           <div class="col-lg-4">
             <p class="description">How does HGT remain in-tact in the face of constant environmental fluctuations and perturbations? What features control HGT specificity (ie, who can receive an HGT signal successfully and who cannot), and how does this ensure functional stability of microbial communities? In other words, what are the design principles that make HGT so ubiquitous and effective? And more importantly, how can we leverage these design principles to create our own networks of genetic communication? These networks could then be applied to diverse microbial applications, including bioremediation, human microbiome engineering, and chemical bioprocessing.</p>
           </div>

          </div>
        </div>

      </div>
    </section><!-- End Services Section -->

    <!-- ======= Services Section ======= -->
    <section class="services section-bg padding-10px">
      <div class="container">

        <div class="row">
          <div class="col-md-6 col-lg-3 d-flex align-items-stretch" data-aos="fade-up">
            <div class="icon-box icon-box-pink">
              <div class="icon"><i class="bx bxs-analyse"></i></div>
              <h4 class="title">Computational Biology</h4>
              <p class="description">Combining mathematical models and biological experiments to understand bacteria</p>
            </div>
          </div>

          <div class="col-md-6 col-lg-3 d-flex align-items-stretch" data-aos="fade-up" data-aos-delay="100">
            <div class="icon-box icon-box-cyan">
              <div class="icon"><i class="bx bx-dna"></i></div>
              <h4 class="title">Genomics</h4>
              <p class="description">Leveraging NGS, RNAseq, and bioinformatics to gain mechanistic insights</p>
            </div>
          </div>

          <div class="col-md-6 col-lg-3 d-flex align-items-stretch" data-aos="fade-up" data-aos-delay="200">
            <div class="icon-box icon-box-green">
              <div class="icon"><i class="bx bx-capsule"></i></div>
              <h4 class="title">Biomolecular Engineering</h4>
              <p class="description">Designing genetic constructs to control bacterial interactions</p>
            </div>
          </div>

          <div class="col-md-6 col-lg-3 d-flex align-items-stretch" data-aos="fade-up" data-aos-delay="200">
            <div class="icon-box icon-box-blue">
              <div class="icon"><i class="bx bx-scatter-chart"></i></div>
              <h4 class="title">Data Science & ML</h4>
              <p class="description">Analyzing data with ML & AI techniques to predict and validate findings</p>
            </div>
          </div>

        </div>

      </div>
    </section><!-- End Services Section -->

    <!-- ======= Service Details Section ======= -->
    <section class="service-details">
      <div class="container">

        <div class="section-title">
          <h2>Research Areas</h2>
        </div>
        <center>The Lopatkin Lab has four main areas of research, spanning multiple disciplines.</center>
        <div class="row">
          <div class="col-md-6 d-flex align-items-stretch" data-aos="fade-up">
            <div class="card">
              <div class="card-img">
                <img src="assets/img/research/cell_quant.png" alt="...">
              </div>
              <div class="card-body">
                <h5 class="card-title"><a href="#">Single-Cell Quantification</a></h5>
                <p class="card-text">HGT is by definition a complex process that can be distilled into individual kinetic parameters (e.g., gene transfer rates, loss rates, etc.). Measuring these parameters with precision and specificity can help us build predictive models that connect sequence-specificity to resulting HGT phenotypes. Projects in the lab are aimed at developing tools to increase the throughput and quantification tools to facilitate large-scale screening of diverse microbes and genetic constructs. These tools include microfluidics, biochemical probes, and computational analysis pipelines. In parallel, results inform mechanistic studies of the underlying molecular biology to better understand why and how unique phenotypes arise. </p>
                <!--<div class="read-more"><a href="#"><i class="bi bi-arrow-right"></i> Read More</a></div>-->
              </div>
            </div>
          </div>
          <div class="col-md-6 d-flex align-items-stretch" data-aos="fade-up">
            <div class="card">
              <div class="card-img">
                <img src="assets/img/research/synth_bio.png" alt="...">
              </div>
              <div class="card-body">
                <h5 class="card-title"><a href="#">Synthetic Biology</a></h5>
                <p class="card-text">Synthetic biology involves arranging DNA in purposeful ways to realize specific logic, much like electronic components are arranged on circuit boards. These “gene circuits” can be introduced into cells to control their behavior. Importantly, the DNA used to build gene circuits consist of either natural constructs (ie genes that are well studied and have known functions), or optimized for specific purposes such as through directed laboratory evolution. In the lab, we use both natural DNA pieces, as well as engineered variants, to create bacterial communities that undergo controllable gene exchange. These synthetically designed gene exchange circuits allows us to study diverse cell-cell interactions and DNA exchange dynamics under highly controlled environments so that we can better understand the critical design aspects of successful HGT events.</p>
                <!--<div class="read-more"><a href="#"><i class="bi bi-arrow-right"></i> Read More</a></div>-->
              </div>
            </div>

          </div>
          <div class="col-md-6 d-flex align-items-stretch" data-aos="fade-up">
            <div class="card">
              <div class="card-img">
                <img src="assets/img/research/amr.png" alt="...">
              </div>
              <div class="card-body">
                <h5 class="card-title"><a href="#">Antibiotic-Mediated Resistance Applications</a></h5>
                <p class="card-text">Antimicrobial resistance represents one of the greatest global medical challenges of the 21st century. Bacteria expressing antibiotic resistance genes (ARGs) can survive lethal antibiotic doses, rendering simple bacterial infections untreatable. ARGs are often encoded on mobile genetic elements that can spread rapidly in bacterial populations through horizontal gene transfer (HGT); indeed, HGT, particularly plasmid conjugation (Fig. 1A), is primarily responsible for the worldwide spread of resistance to antibiotic drugs. Understanding the mechanisms governing HGT is critical to both prolong the shelf life of existing drugs, and to develop new therapeutic approaches. Ongoing research focuses on understanding the role of HGT in the spread of antibiotic resistance, and just as importantly, investigating the evolutionary consequences of interfering in HGT as a therapeutic strategy.</p>
                <!--<div class="read-more"><a href="#"><i class="bi bi-arrow-right"></i> Read More</a></div>-->
              </div>
            </div>
          </div>
          <div class="col-md-6 d-flex align-items-stretch" data-aos="fade-up">
            <div class="card">
              <div class="card-img">
                <img src="assets/img/research/ml.png" alt="...">
              </div>
              <div class="card-body">
                <h5 class="card-title"><a href="#">Computational Platforms for Prediction and Understanding</a></h5>
                <p class="card-text">Given the diversity, complexity, and quantity of data necessary to study and rigorously quantify HGT, machine learning (ML) approaches present a natural avenue to predict both broad ecological trends and molecular underpinnings. However, standard “black box” ML approaches are inherently unaware of biological mechanisms, and may form biologically inappropriate or misleading statistical relationships. To overcome this weakness, the lab is focused on developing “white-box” ML tools that incorporate both large ‘omics datasets and underlying mechanistic understanding, while constraining predictions biologically. In addition to ML, this arm of the lab involves both mathematical modeling approaches as well as in-house developed bioinformatic pipelines.</p>
                <!--<div class="read-more"><a href="#"><i class="bi bi-arrow-right"></i> Read More</a></div>-->
              </div>
            </div>
          </div>
        </div>

      </div>
    </section><!-- End Service Details Section -->

  </main><!-- End #main -->

 <!-- ======= Footer ======= -->
  <footer id="footer" data-aos="fade-up" data-aos-easing="ease-in-out" data-aos-duration="500">

    <div class="footer-top">
      <div class="container">
        <div class="row">

           <div class="col-lg-3 col-md-6 footer-links">
            <h4>Affiliations</h4>
            <ul>
              <li><i class="bx bx-chevron-right"></i> <a href="https://www.hajim.rochester.edu/che/people/faculty/lopatkin-allison/index.html" target="_blank"UR, <br>Dept. of Chemical Engineering</a></li>
              <li><i class="bx bx-chevron-right"></i> <a href="https://www.urmc.rochester.edu/microbiology-immunology/people/faculty.aspx" target="_blank">URMC, <br>Microbiology & Immunology</a></li>
            </ul>
          </div>

          <div class="col-lg-3 col-md-6 footer-contact">
            <h4>Contact Us</h4>
            <p>
              313 Gavett Hall<br>
              Univeristy of Rochester<br>
              Rochester, NY 14627<br><br>
              <strong>Email:</strong> <a href="mailto:lopatkinlab@gmail.com">lopatkinlab@gmail.com</a><br>
            </p>
          </div>

          <div class="col-lg-3 col-md-6 footer-info">
            <h3>Around the Web</h3>
            <p>We're always adding new content to our Instagram, YouTube, Twitter and TikTok accounts!</p>
            <div class="social-links mt-3">
              <a href="https://twitter.com/LopatkinLab" class="twitter" target="_blank"><i class="bx bxl-twitter"></i></a>
              <a href="https://www.tiktok.com/@lopatkinlab" class="facebook" target="_blank"><i class="bx bxl-tiktok"></i></a>
              <a href="https://instagram.com/lopatkinlab" class="instagram" target="_blank"><i class="bx bxl-instagram"></i></a>
              <a href="https://www.youtube.com/channel/UCgWH0nCqBdG-9k0JcLFZS7w" target="_blank"><i class="bx bxl-youtube"></i></a>
              <a href="https://github.com/ajlopatkin" class="linkedin" target="_blank"><i class="bx bxl-github"></i></a>
            </div>
          </div>

        </div>
      </div>
    </div>

    <div class="container">
      <div class="credits">
        <!-- All the links in the footer should remain intact. -->
        <!-- You can delete the links only if you purchased the pro version. -->
        <!-- Licensing information: https://bootstrapmade.com/license/ -->
        <!-- Purchase the pro version with working PHP/AJAX contact form: https://bootstrapmade.com/free-bootstrap-template-corporate-moderna/ -->
        Designed by <a href="https://bootstrapmade.com/">BootstrapMade</a>
      </div>
    </div>
  </footer><!-- End Footer -->

  <a href="#" class="back-to-top d-flex align-items-center justify-content-center"><i class="bi bi-arrow-up-short"></i></a>

  <!-- Vendor JS Files -->
  <script src="assets/vendor/purecounter/purecounter.js"></script>
  <script src="assets/vendor/aos/aos.js"></script>
  <script src="assets/vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
  <script src="assets/vendor/glightbox/js/glightbox.min.js"></script>
  <script src="assets/vendor/isotope-layout/isotope.pkgd.min.js"></script>
  <script src="assets/vendor/swiper/swiper-bundle.min.js"></script>
  <script src="assets/vendor/waypoints/noframework.waypoints.js"></script>
  <script src="assets/vendor/php-email-form/validate.js"></script>

  <!-- Template Main JS File -->
  <script src="assets/js/main.js"></script>

</body>

</html>
